test/fixtures/templates/deliveries/_form.html.erb

```diff
@@ -2,7 +2,7 @@
   form_for(form, class: 'form-horizontal') do
     div class: 'form-group' do
       label :customer
-      input_text :customer, class: 'form-control', placeholder: 'Customer', name: nil
+      input_text :customer, class: 'form-control', placeholder: 'Customer', name: nil, required: true
```

Comment:
Sure, i'll remove from integration. It'll need adding to a few tests as it's valid for `<input>, <select>, <textarea>`